lib/ttnt/testtask.rb

```diff
@@ -0,0 +1,35 @@
+module TTNT
+  class TestTask
+    @@instances = []
+
+    def self.instances
+      @@instances
+    end
+
+    ATTRIBUTES = %i(
+      name
+      libs
+      pattern
+      options
+      test_files
+      warning
+      loader
+      ruby_opts
+      description
+      pattern
+    ).freeze
+
+    ATTRIBUTES.each do |attr|
+      attr_reader attr
+    end
+
+    def initialize(rake_test_task)
+      ATTRIBUTES.each do |attr|
```

Comment:
Wow, this is so smart. Thank you! :smile:
I changed to this approach here: https://github.com/Genki-S/ttnt/commit/28a1f289eb5de5af7e60f22d38a561ff31d3cf92

> you put pattern two times while verbose is missing

oops... thanks!

> I don't know what the respond_to? call is for ; maybe the missing test_files attribute reader ?

Exactly! I left the `respond_to?` check since it's possible someone can set new instance variable to an object of `Rake::TestTask`. (though I'm not sure if I should care about such a situation... I'm new to this magical Ruby world where you can access instance variable of any object :P)